openstf/model/train.py

```diff
@@ -160,30 +243,93 @@ def train_model_for_specific_pj(
         model_trainer.confidence_interval,
     )
     train_predict = predictor.make_forecast(
-        train_data.iloc[:, 1:].drop("Horizon", axis=1, errors="ignore")
+        split_input_data["train_data"]
+        .iloc[:, 1:]
+        .drop("Horizon", axis=1, errors="ignore")
     )
     validation_predict = predictor.make_forecast(
-        validation_data.iloc[:, 1:].drop("Horizon", axis=1, errors="ignore")
+        split_input_data["validation_data"]
+        .iloc[:, 1:]
+        .drop("Horizon", axis=1, errors="ignore")
     )
     test_predict = predictor.make_forecast(
-        test_data.iloc[:, 1:].drop("Horizon", axis=1, errors="ignore")
+        split_input_data["test_data"]
+        .iloc[:, 1:]
+        .drop("Horizon", axis=1, errors="ignore")
     )
 
-    context.perf_meter.checkpoint("predicting")
+    return {
+        "train_predict": train_predict,
+        "validation_predict": validation_predict,
+        "test_predict": test_predict,
+    }
+
+
+def create_evaluation_figures_for_specific_pj(
```

Comment:
Just wondering if this should actually be part of the pipeline. Do you always want to run such a function?